### ISSUE TITLE
Unclear description of exporter

### ISSUE THREAD

**Issue by sanjaypujare** [census-instrumentation/opencensus-website#693]
The following description of an exporter:

>Exporters democratize consumption of traces and metrics by any backend that can consume them. They are the vendor agnosticity that OpenCensus touts.

is unclear because of the unnecessarily grandiose language. Can we say something like:

> An exporter can export traces and metrics to any backend that is capable of consuming them. The exporter can do it without requiring any changes in the backend's interface which makes OpenCensus truly vendor agnostic.

**Comment by odeke-em** [comment]
Thank you @sanjaypujare for reporting this and welcome to the OpenCensus project!

Awesome, thank you for the simplification/reduction! That sentence is like that because of a previous fear of being too wordy, not an intentionally grandiose wording. Would you like to send a PR or would you like me to send it?

Thank you.

**Comment by sanjaypujare** [comment]
It will be great if you could send it. Thank you!

**Comment by odeke-em** [comment]
Awesome, I've mailed PR https://github.com/census-instrumentation/opencensus-website/pull/694, please help me take a look and provide a code review. Thank you.
